src/main/java/duke/MainUi.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+public class MainUi extends Ui {
```

Comment:
Include a Javadoc for your classes as it is unclear what each class does.

Reply:
Without constructor JavaDoc, I believe that is necessary. 👍 Good suggestion.